src/test/java/com/andersen/ticketservice/TicketServiceTest.java

```diff
@@ -0,0 +1,234 @@
+package com.andersen.ticketservice;
+
+import lesson10.exception.TicketNotFoundException;
+import lesson10.model.Ticket;
+import lesson10.repository.TicketRepository;
+import lesson10.service.TicketService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.dao.InvalidDataAccessApiUsageException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.times;
+
+
+public class TicketServiceTest {
+
+    private static TicketService ticketService;
+    private static TicketRepository ticketRepository;
+
+    @BeforeAll
+    public static void init() {
+        ticketRepository = mock(TicketRepository.class);
+        ticketService = new TicketService(ticketRepository);
+    }
+
+    @Test
+    public void createTicket_ValidTicket_Saved() {
+        Ticket ticketToSave = new Ticket();
```

Comment:
In case when you need new object for each test you could move this creation to separated method and use @BeforeEach annotation